token/program-2022/src/extension/confidential_transfer/verify_proof.rs

```diff
@@ -460,6 +468,8 @@ pub fn verify_transfer_with_fee_proof(
             .maximum_fee
             .into();
 
+        // check consistency of the transfer fee parameters in the mint extension with what were
+        // used to generate the zkp
```

Comment:
nit: can you just add: ", which is not checked in the `From` implementation"? that way we're sure

Reply:
Yep, good idea